testgrid/config.yaml

```diff
@@ -6272,6 +6307,7 @@ dashboard_groups:
 - name: sig-cluster-lifecycle
   dashboard_names:
   - sig-cluster-lifecycle-all
+  - sig-cluster-lifecycle-kubeadm
```

Comment:
so something about -all vs -kubeadm.

in my opinion we should make -all only hold jobs that don't exist in other dashboards already.

not a big issue, but it makes us duplicate jobs.
the testgrid dashboard titles do light red if there are failing tests under that dashboard, so navigating them is not a problem.

Reply:
agreed